src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests.csproj

```diff
@@ -34,10 +34,12 @@
     <Compile Include="ProcessStandardConsoleTests.cs" />
     <Compile Include="ProcessStartInfoTests.cs" />
     <Compile Include="ProcessStreamReadTests.cs" />
+    <Compile Condition="'$(TargetGroup)' == 'netcoreapp' or '$(TargetGroup)' == 'uap'" Include="ProcessStreamReadTests.netcoreapp2.1.cs" />
```

Comment:
Could you please move this two Conditioned Compile items to a separate ItemGroup and condition the ItemGroup instead? That makes it much cleaner to read.